library/jsonviewer/src/main/java/org/billcarsonfr/jsonviewer/ValueItem.kt

```diff
@@ -77,8 +77,7 @@ internal abstract class ValueItem : EpoxyModelWithHolder<ValueItem.Holder>() {
         ) {
             if (copyValue != null) {
                 val menuItem = menu?.add(R.string.copy_value)
-                val clipService =
-                        v?.context?.getSystemService(Context.CLIPBOARD_SERVICE) as? ClipboardManager
+                val clipService = v?.context?.getSystemService<ClipboardManager>()
```

Comment:
Maybe here we could use [this function](https://github.com/vector-im/element-android/blob/5efe1f4bd8e065f9dc33d46edc53e38d2a9ba85f/vector/src/main/java/im/vector/app/core/utils/SystemUtils.kt#L88)

Reply:
no, we are in a submodule. Or this fun must be move to a deeper module in the hierarchy.